moonshot/integrations/cli/redteam/redteam.py

```diff
@@ -88,3 +103,26 @@ def do_delete_context_strategy(self, args: argparse.Namespace) -> None:
     @cmd2.with_argparser(delete_attack_module_args)
     def do_delete_attack_module(self, args: argparse.Namespace) -> None:
         delete_attack_module(args)
+
+    @cmd2.with_argparser(add_bookmark_prompt_args)
+    def do_bookmark_prompt(self, args: argparse.Namespace) -> None:
+        bookmark_prompt(args)
+
+    @cmd2.with_argparser(use_bookmark_args)
+    def do_use_bookmark(self, args: argparse.Namespace) -> None:
+        use_bookmark(args)
+
+    @cmd2.with_argparser(delete_bookmark_prompt_args)
+    def do_delete_bookmark(self, args: argparse.Namespace) -> None:
+        delete_bookmark(args)
+
+    def do_list_bookmarks(self, _: cmd2.Statement) -> None:
+        list_bookmarks()
+
+    @cmd2.with_argparser(view_bookmark_prompt_args)
```

Comment:
can we change this to view_bookmark_args from view_bookmark_prompt_args

Reply:
updated